src/workloads/query/GroupLikeDistinct.yml

```diff
@@ -0,0 +1,169 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests performance of $group queries with distinct like semantics, meaning they will
+  execute with a DISTINCT_SCAN. Currently this is supported for $group + $sort and only with the
+  $first and $last accumulators. In PM-3163 this will be extended to $top, $topN and $bottom,
+  $bottomN accumulators so we are including those here in order to see performance changes once that
+  is implemented.
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 2000
```

Comment:
This is to avoid hitting the `maxPoolSize` which causes this error: 
`Error in parallel execution. First exception's what(): Failed to acquire an entry from the client pool.`